packages/wonder-blocks-form/src/components/checkbox-core.tsx

```diff
@@ -50,13 +68,31 @@ export default class CheckboxCore extends React.Component<ChoiceCoreProps> {
             "data-test-id": testId,
         } as const;
 
+        const checkboxIcon = (
+            <Icon
+                color={disabled ? offBlack32 : white}
+                icon={checked ? checkPath : indeterminatePath}
+                size="small"
+                style={sharedStyles.checkboxIcon}
+            />
+        );
+
+        const ariaChecked = mapCheckedToAriaChecked(checked);
+
+        // TODO (nicolemejia) - We're not using `.indeterminate` on
+        // `StyledInput` because `addStyle()` doesn't support ref forwarding. We
+        // should consider adding it in the future to round out the component to
+        // align with input checkbox indeterminate property. Read more here:
+        // https://www.w3schools.com/jsref/prop_checkbox_indeterminate.asp
         return (
             <React.Fragment>
                 <StyledInput
                     {...sharedProps}
                     type="checkbox"
+                    role="checkbox"
```

Comment:
**suggestion:** you can get rid of this `role` as it is implicitly defined by the `input[checkbox]` element.

```suggestion         
```